ansibullbot/triagers/plugins/file_deprecations.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+
+import datetime
+from distutils.version import LooseVersion
+
+
+def get_deprecation_facts(issuewrapper, meta, versionindexer):
+
+	# https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/docker/_docker.py#L21-L24
+    #deprecated:
+    #   removed_in: "2.4"
+    #   why: Replaced by dedicated modules.
+    #   alternative: Use M(docker_container) and M(docker_image) instead.
+
+    dmeta = {
+        'is_deprecated': False,
+        'deprecation_action': None
+    }
+
+    iw = issuewrapper
+
+    if len(meta['component_matches']) != 1:
+        return dmeta
+
+    cm = meta['component_matches'][0]
+    if cm.get('deprecated'):
+        dmeta['is_deprecated'] = True
+
+        if 'removed_in' not in cm['deprecation_info']:
```

Comment:
This should be enforced by `schema.py`, though makes sense as a safety check. Does ansibull bot have the ability to warn?